src/lib/workspace/Subsetting/SubsetDownloadModal.tsx

```diff
@@ -706,21 +706,32 @@ export default function SubsetDownloadModal({
 
   // ~18px (round to 20px) per character for medium title size
   const maxStudyNameLength = Math.floor(modalHeaderWidth / 20);
-  const studyName =
-    studyRecord.displayName.length > maxStudyNameLength ? (
-      <span title={studyRecord.displayName}>
-        {safeHtml(
-          studyRecord.displayName.substring(0, maxStudyNameLength - 2) + '...'
-        )}
+  const regexAllTags = /<.*?>/g;
+  const studyNameNoTags = studyRecord.displayName.replaceAll(regexAllTags, '');
+  let studyNameElement: JSX.Element;
+
+  if (studyNameNoTags.length > maxStudyNameLength) {
+    const tagCharCount =
+      studyRecord.displayName.length - studyNameNoTags.length;
+    const studyNameTrunc = studyRecord.displayName.substring(
+      0,
+      maxStudyNameLength + tagCharCount - 2
+    );
+    const regexUnclosedTag = /<(?!.*>).*/;
+    const studyNameNoUnclosedTag = studyNameTrunc.replace(regexUnclosedTag, '');
+    studyNameElement = (
+      <span title={studyNameNoTags}>
+        {safeHtml(studyNameNoUnclosedTag + '...')}
       </span>
```

Comment:
FYI, you can pass props to `safeHtml`. It defaults to using a `<span>` tag to wrap the content passed in as the first argument.
```suggestion
      {safeHtml(studyNameNoUnclosedTag + '...', { title: studyNameNoTags })}
```